suite-native/atoms/src/Radio.tsx

```diff
@@ -64,8 +64,6 @@ export const Radio = ({
         <TouchableOpacity
             disabled={isDisabled}
             onPress={() => onPress(value)}
-            accessibilityRole="radio"
```

Comment:
Why did you remove these two lines? I don't know accessibility API very well, so this is not request for change, I'm just curious.

Reply:
Radio is always used with a wrapper that's triggering the value switch. So in select for example, there is `SelectItem` which triggers the radio change. I added the `accessibilityRole` to the wrapper component because it's needed there and since it's there, I believe it is redundant here. Furthermore, you would only get the role, but no label, which I added to the wrapper as well, so I think it's better this way. Radio is currently only used in `SelectItem` and `SelectableListItem` (which was never used so I deleted it).

I'm a little bit unsure about this but believe this is better. If you agree I would stick with this and see if google reports it or not.